### PR TITLE
🌐 Add Indonesian translation for `docs/id/docs/tutorial/first-steps.md`

### PR DESCRIPTION
**Pull request by gerry-sabar**
Hi, I notice the Indonesian translation for this section isn't maintained. Going forward I'd love to bring more contribution for Indonesian translation starting from this file @tiangolo

### CODE REVIEW

**Review by RyaWcksn — APPROVED** [review]
LGTM 🚀

**Comment by ilhamfadillah** [comment]
It's Good

**Review by ilhamfadillah — COMMENTED** [review]
It's Good

**Review by ilhamfadillah — APPROVED** [review]
its ok

File: docs/id/docs/tutorial/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+Di kondisi ini, ini adalah sebuah fungsi `async`.
+
+---
+
+Anda bisa mendefinisikan fungsi ini sebagai fungsi normal daripada `async def`:
+
+{* ../../docs_src/first_steps/tutorial003.py hl[7] *}
+
+/// catatan
```
Comment (alejsdev) [inline docs/id/docs/tutorial/first-steps.md:310]: This is not displaying properly. Can you check it, please?
You can check the translation guidelines if you have any questions. Thanks!

Comment (gerry-sabar) [reply docs/id/docs/tutorial/first-steps.md]: Hi @alejsdev thanks for the review. I have fixed improper code for text boxes so now all text boxes are displayed properly

**Comment by alejsdev** [comment]
Thank you for the help! @gerry-sabar :rocket:

**Pull request merged**